README.md

```diff
@@ -1,52 +1,49 @@
-<img src="https://www.kubeflow.org/images/logo.svg" width="100">
+# Kubeflow
 
-[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
+[![Join Kubeflow Slack](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels)
 [![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/kubeflow/badge)](https://clomonitor.io/projects/cncf/kubeflow)
+[![OpenSSF Best Practices](https://www.bestpractices.dev/projects/2660/badge)](https://www.bestpractices.dev/projects/2660)
 
-The Kubeflow makes AI/ML on Kubernetes simple, portable and scalable.
+<img src="./logo/icon.svg" width="120">
 
----
+## About Kubeflow
 
-## Documentation
+[Kubeflow](https://www.kubeflow.org/) makes artificial intelligence and machine learning simple, portable, and scalable.
+We are an _ecosystem_ of [Kubernetes](https://kubernetes.io/) based components for each stage in the [AI/ML Lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle) with support for best-in-class open source [tools and frameworks](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem).
 
-Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
+Please refer to the official [documentation](https://www.kubeflow.org/docs/) for more information.
 
 ## Kubeflow Components
 
-The Kubeflow ecosystem is composed of multiple open-source projects for each stage in
-[the ML lifecycle](https://www.kubeflow.org/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
+The [Kubeflow Ecosystem](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem) is composed of several projects known as [Kubeflow Components](https://www.kubeflow.org/docs/components/).
```

Comment:
Do we want to mention `open source project` ?
```suggestion
The [Kubeflow Ecosystem](https://www.kubeflow.org/docs/started/architecture/#kubeflow-ecosystem) is composed of several open source projects known as [Kubeflow Components](https://www.kubeflow.org/docs/components/).
```

Reply:
Personally, I think this is redundant, given its pretty clearly open source (and we are on GitHub).

It makes the sentence harder to read (longer), but if others disagree, we can add it.

Reply:
Sure, no problem.